readme.md

```diff
@@ -31,43 +31,20 @@ The Salesforce Mobile SDK for iOS requires iOS 11.0 or greater.  The install.sh
-from Apple’s unified logging system. See [iOS Compiler-Level Logging](https://developer.salesforce.com/docs/atlas.en-us.noversion.mobile_sdk.meta/mobile_sdk/analytics_logging_ios.htm).
-
-**Miscellaneous Changes**
-- We’ve simplified `AppDelegate` initialization logic by decoupling login from SDK initialization. You’re no longer required to listen to
-and handle Mobile SDK launch events.
-- Advanced authentication is now allowed by default. The type of authentication used by a Mobile SDK app can be configured only
-through My Domain settings.
-- As recommended by Apple, we’ve updated iOS advanced authentication to use `SFAuthenticationSession` instead of
-`SFSafariViewController`. This notice is informational only and does not require any action on your part.
-- Mobile SDK apps now support both Face ID and Touch ID.
-- We’ve updated and improved the app passcode dialog box.
-  
-**iOS Version Updates**
-- iOS minimum version (deployment target): iOS 11
-- iOS base SDK version: iOS 12
-- Xcode minimum version: 10
-  
-**Deprecation**
-- `PromiseKit` is no longer a dependency of Mobile SDK. Instead, you can use standard delegates or blocks to handle asynchronous calls.
-- `SFSDKLogger` is now deprecated in Mobile SDK apps. Use `os_log()` instead.
-Check http://developer.force.com/mobilesdk for additional articles and tutorials.
+### What's New in 7.1
+
+- We’ve improved support for sending unauthenticated REST requests to external endpoints. Mobile SDK now provides a shared global instance of its REST client. This REST client doesn’t require OAuth authentication and is unaware of the concept of user. Native apps can use it to send custom unauthenticated requests to non-Salesforce endpoints before or after the user logs in to Salesforce.
+- We’ve added support for using biometric input to supply application passcodes.
+- We’ve updated our Swift template app to Swift version 5.0 and Xcode version 10.2.
+- Mobile SDK for iOS raises its master key security to use Secure Enclave on devices that support it. We’ve also strengthened our master key encryption to use a 256-bit elliptic curve cryptography (ECC) private key.
+- For profiling how Mobile SDK operations affect an app’s runtime performance, we’ve added signposts to Mobile SDK libraries.
+
+**What’s New in SmartSync Data Framework for Mobile SDK 7.1**
+- SmartSync Data Framework now supports a batch sync up target that uses the Salesforce Composite API for uploading groups of up to 25 records per call.
+- New methods allow native apps to stop and restart in-flight sync operations. To reflect the new sync state, we’ve added a stopped sync status.
+- Swift `syncUp` and `syncDown` methods that take a sync name, both `reSync` methods, and the legacy `cleanResyncGhosts` method now can throw exceptions.
+- We’ve updated iOS methods to make error handling more consistent between platforms.
+- You can now call `cleanResyncGhosts` with a sync name.
```

Comment:
We also need a `Deprecation` section - I know some APIs were deprecated in `SmartSync` and even in `SalesforceSDKCore`.

Reply:
Is there a list of core deprecations? I only know the SmartSync methods.